m/processing_tools/gridGliderData.m

```diff
@@ -306,26 +306,31 @@
   fprintf('  number of variables   : %d\n', num_variables);
   data_grid_variables = nan(num_casts, num_levels, num_variables);
   for cast_idx = 1:num_casts
-    cast_select = (profile == cast_idx);
-    cast_lat = latitude(cast_select);
-    cast_lon = longitude(cast_select);
-    cast_depth = depth(cast_select);
-    cast_time = time(cast_select);
-    cast_variables = variables(cast_select, :);
-    data_grid.time(cast_idx) = nanmean(cast_time);
-    data_grid.latitude(cast_idx) = nanmean(cast_lat);
-    data_grid.longitude(cast_idx) = nanmean(cast_lon);
-    if ~isempty(cast_variables) % Speed up when there are no variables.
-      data_grid_variables(cast_idx, :, :) = ...
-        cell2mat(arrayfun(@(d) nanmean(cast_variables(abs(cast_depth-d)<=0.5*depth_resolution, :), 1), ...
-                          depth_range(:), 'UniformOutput', false));
-    end
-  end
-  % Move binned variable data to output struct.
-  for variable_name_idx = 1:num_variables
-    variable_name = variable_name_list{variable_name_idx};
-    data_grid.(variable_name) = data_grid_variables(:, :, variable_name_idx);
-  end
+      cast_select = (profile == cast_idx);
+      cast_lat = latitude(cast_select);
+      cast_lon = longitude(cast_select);
+      cast_depth = depth(cast_select);
+      cast_time = time(cast_select);
+      cast_variables = variables(cast_select, :);
+      data_grid.time(cast_idx) = nanmean(cast_time);
+      data_grid.latitude(cast_idx) = nanmean(cast_lat);
+      data_grid.longitude(cast_idx) = nanmean(cast_lon);
+
```

Comment:
Unintentional indentation size changes here maybe? It should to 2 spaces per indentation level.